check261

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Prowler - the handy cloud security tool (copyright 2019) by Toni de la Fuente
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+CHECK_ID_check261="26.1"
+CHECK_TITLE_check261="[check261] CodeBuild Project last invoked greater then 90 days"
+CHECK_SCORED_check261="NOT_SCORED"
+CHECK_CIS_LEVEL_check261="LEVEL1"
+CHECK_SEVERITY_check261="High"
+CHECK_ASFF_TYPE_check261="Software and Configuration Checks/Industry and Regulatory Standards/CIS AWS Foundations Benchmark"
+CHECK_ALTERNATE_check101="check261"
+CHECK_SERVICENAME_check261="codebuild"
+CHECK_RISK_check261='The CodeBuild projects are very old and can be checked if they are being used currently'
+CHECK_REMEDIATION_check261=''
+CHECK_DOC_check261=''
+CHECK_CAF_EPIC_check261='CODEBUILD'
+
+check261(){
+    textInfo "Looking for all build projects with last build invocation greater then 30 days in all regions"
```

Comment:
You can only use `textInfo`, `textPass` and `textFail` to log check results whether it's a fail, a pass or some information within a check (ex. lack of permission)